config/config.go

```diff
@@ -530,12 +530,13 @@ func DefaultFuzzConnConfig() *FuzzConnConfig {
 
 // MempoolConfig defines the configuration options for the Tendermint mempool
 type MempoolConfig struct {
-	RootDir   string `mapstructure:"home"`
-	Recheck   bool   `mapstructure:"recheck"`
-	Broadcast bool   `mapstructure:"broadcast"`
-	WalPath   string `mapstructure:"wal_dir"`
-	Size      int    `mapstructure:"size"`
-	CacheSize int    `mapstructure:"cache_size"`
+	RootDir          string `mapstructure:"home"`
+	Recheck          bool   `mapstructure:"recheck"`
+	Broadcast        bool   `mapstructure:"broadcast"`
+	WalPath          string `mapstructure:"wal_dir"`
+	MaxTxs           int    `mapstructure:"max_txs"`
+	MaxTxsTotalBytes int64  `mapstructure:"max_txs_total_bytes"`
+	CacheSize        int    `mapstructure:"cache_size"`
```

Comment:
CacheMaxTxs?

Reply:
In case of the cache, we do allocate memory, so it's completely fair to name the config parameter `CacheSize`